tests/Feature/SettingsPageTest.php

```diff
@@ -0,0 +1,370 @@
+<?php
+/**
+ * WP SEO Tests: Tests for the settings page, the settings fields, and settings section help text.
+ *
+ * @package wp-seo
+ */
+
+namespace Alley\WP\WP_SEO\Tests\Feature;
+
+use Alley\WP\WP_SEO\Tests\TestCase;
+use Mantle\Testing\Utils;
+use WP_SEO_Settings;
+use WP_SEO;
+use Mantle\Testing\Concerns\Admin_Screen;
+
+class SettingsPageTest extends TestCase {
+	use Admin_Screen;
+
+	/**
+	 * Make sure we have a settings page.
+	 */
+	function test_add_options_page() {
```

Comment:
While this is still valid PHP, its common practice to explicitly set method visibility for all methods.

```suggestion
	public function test_add_options_page() {
```